doc/can-route.md

```diff
@@ -261,8 +314,8 @@ route.start();
 
 ## Changing the route
 
-Typically, you don’t set `location.hash`
-directly. Instead, you can change properties on can-route
+Typically, you don’t set `location.hash` directly.
+Instead, you can change properties on can-route
```

Comment:
show using can-stache-route-helpers instead.

Reply:
`route.data.assign( { } )` to overwrite, but not delete properties 

`route.data.update( {})` overwrite AND delete properties